redis/README.md

```diff
@@ -54,45 +54,45 @@ WARNING:
 
 # What is Redis?
```

Comment:
Any changes in `redis/README.md` should be left out of the PR, it'll be automatically regenerated after merge.